static/js/util/sorting.js

```diff
@@ -1,10 +1,12 @@
 // @flow
```

Comment:
`resumeOrder` is what this ordering was referred to on the issue, I just went with that.